### ISSUE TITLE
Increase the contrast of the Dock Icon

### ISSUE THREAD

**Issue by stevendaniels** [git-up/GitUp#216]
Thanks for gitup. It's a great app.

The contrast of the icon is pretty low.  When switching between applications (CMD+Tab),  it almost looks like a white square.

**Comment by sirnacnud** [comment]
I'm not for sure what the icon is supposed to represent or signify, but it doesn't stand out when compared to other apps. It looks like some kind of default app icon.

**Comment by muescha** [comment]
i would also vote for a better icon. some icon ideas are in #396

**Comment by sesam** [comment]
Yep, the dock icon is completely impossible to see -- just a white blur in most lighting conditions.
#good-first-issue
